Update sql-insights-overview.md (#9665)
MicrosoftDocs/sql-docs: azure-sql/database/sql-insights-overview.md

For more instructions, see [Enable SQL Insights (preview)](sql-insights-enable.md) and [Troubleshoot SQL Insights (preview)](sql-insights-troubleshoot.md).

> **Note:** SQL insights are not supported for SQL Server On Prem Instances.

Reviewer: ```suggestion
> [!NOTE]
> SQL Insights (preview) isn't supported for SQL Server on-premises instances.
```
